### DISCUSSION TITLE
Is LNP Node suitable for micropayments?

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/presentations#41] [Q&A]
### TL;DR
- any LN node is suitable for micropayments and LNP Node is not an exception
- LNP Node will also be interoperable with [Lightspeed protocol](https://github.com/dr-orlovsky/notes/blob/master/lightspeed_microtx.md) for nanopayments.

### TS;WM

Any LN node is suitable for micropayments, it’s a feature of the protocol, not a feature of the node. So sure, the LNP Node is suitable. There is one distinction though. 

We have a protocol called [Lightspeed](https://github.com/dr-orlovsky/notes/blob/master/lightspeed_microtx.md), which is what we call, a protocol for **nano payments**. The difference from micropayments is that with micropayments you have to do about at least 3 network requests + create at least 6 signatures (in fact, the number of signatures can be much higher) + the time and a lot of computational resource to create all that. [LSAT protocol](https://github.com/lightninglabs/LSAT), announced fairy recently by Lightning Labs, improves this situation - it allows you to reduce the number or network requests but it does not affect the number of signatures that have to be created by the node. Lightspeed protocol that we have proposed allows you to reduce the number of signatures to 0. Basically, you can make a payment without creating a signature. It works nicely with RGB also, where you can pay not only with bitcoin but also make nanopayments with RGB assets, USDT etc. Also by that you can create services that would accept payments, say, per second of video streaming or per meter of a road you use, meaning that the server running such a node will have a load several orders of magnitude smaller compared to existing numbers. If you have, for instance, a million clients connected to a server, you simply wouldn’t be able to work through creating 6 signatures per micropayment thus the load will stop the service from working. This is why reducing the amount of signatures is highly important. 

To conclude,  Lightspeed will be really important when we have adoption of micropayments and LNP Node is planned to support Lightspeed protocol.
